lib/AST/ASTPrinter.cpp

```diff
@@ -5050,6 +5098,13 @@ void PrintAST::visitErasureExpr(ErasureExpr *expr) {
 }
 
 void PrintAST::visitKeyPathExpr(KeyPathExpr *expr) {
```

Comment:
`KeyPathExpr` is also used to represent Obj-C key paths, e.g `#keyPath(...)`, you'll want to check `expr->isObjC()` for that